sdk/src/main/java/com/mapbox/maps/ViewAnnotationManagerImpl.kt

```diff
@@ -212,9 +248,14 @@ internal class ViewAnnotationManagerImpl(
     // add and reposition new and existed views
     positionDescriptorCoreList.forEach { descriptor ->
       annotationMap[descriptor.identifier]?.let { annotation ->
+        // update layout params explicitly if user has specified concrete width or height
         annotation.viewLayoutParams.apply {
-          width = descriptor.width
-          height = descriptor.height
+          if (annotation.measuredWidth == -1) {
```

Comment:
please, replace `-1` with some constant

Reply:
yeah, thanks for pointing it out, indeed code looks much more readable.